wiki/Project_Loved/id.md

```diff
@@ -13,7 +8,8 @@ Beatmap dikategorikan menjadi Loved setelah melewati proses nominasi dan pemungu
 
 Anggota Project Loved Team yang bergelar **kapten** secara teratur mencalonkan beatmap untuk mode tertentu. Hampir selalu, set yang dicalonkan berasal dari kiriman komunitas osu!. Formulir dan daftar pengajuan Loved saat ini dapat diakses melalui tautan berikut:
 
-- [Formulir osu!/osu!taiko/osu!catch](https://docs.google.com/forms/d/e/1FAIpQLSdbgHOVqMF8wQQKSdddW1JhC10ff6C7fb4JbEW7PBQTn9gAqg/viewform)
+- [Formulir osu!/osu!catch](https://docs.google.com/forms/d/e/1FAIpQLSdbgHOVqMF8wQQKSdddW1JhC10ff6C7fb4JbEW7PBQTn9gAqg/viewform)
+- [Formulir osu!taiko](https://docs.google.com/forms/d/e/1FAIpQLSclPWyjFByhHP45AtKD49y0RSl1TK5UOzD4dVdvjfJJQ1aCXQ/viewform)
 - [Formulir osu!mania](https://docs.google.com/forms/d/e/1FAIpQLSeaGfoQNGMqw4qQcqRPItUZILh2fGwJR6ly6cZNY9OWPXkFhw/viewform)
 - [Daftar osu!/osu!taiko/osu!catch](https://docs.google.com/spreadsheets/d/1HgHwtO3kIzT8R4ocEJMZTosADrGJRJOFL-TZI97tZS4/edit)
```

Comment:
`Daftar osu!/osu!taiko/osu!catch` should be changed to `Daftar osu!/osu!catch`. Don't forget to add `Daftar osu!taiko` to the list (also the link).

Reply:
fml